### DISCUSSION TITLE
Allow arrow navigation for selecting new block type

### DISCUSSION THREAD

**Discussion by Duncank** [codex-team/editor.js#1918] [Ideas]
When adding a new block with keyboard shortcuts (tab in an empty block), it looks like it is possible to select other blocks by using the arrow keys, but pressing an arrow key jumps the cursor to the next / previous paragraph, instead of selecting the next block-type.

![image](https://user-images.githubusercontent.com/719365/58941153-7bd82200-877b-11e9-9fc9-5949ff8aac41.png)

Using 'tab' repeatedly does move between selected block-types, but arrow keys would be a nice usability update here.

**Comment by neSpecc** [comment]
Good idea, will be implemented in next releases.
